### DISCUSSION TITLE
LT keeps requiring me to recreate pairing phrase roughly every 2-3 days.

### DISCUSSION THREAD

**Discussion by tkhumush** [lightninglabs/lightning-terminal#849] [Troubleshooting]
I keep losing my session, failing to log in and leading to restart my node and repair again. Any thoughts on what might be happening or how I can improve stability?

**Comment by guggero** [comment]
Can you post some logs from your node (and also from the browser console that opens with F12) please? Most useful would be the logs from when the connection fails and before you have to re-create it.

**Reply by mattius459** [reply]
This happens to me every 2-3 days as well. I'm using voltage.
